src/Menu/NotificationManager.php

```diff
@@ -184,6 +185,25 @@ public function display_aggregated_notification_pill(): void {
 		}
 	}
 
+	/**
+	 * Filter the notification count for the admin menu.
+	 *
+	 * This method checks the user's persisted preferences to determine if there are any recommendations.
+	 * If there are recommendations, it increments the count by 1; otherwise, it returns 0.
+	 *
+	 * @return int The updated notification count, which is either 1 (if there are recommendations) or 0 (if there are no recommendations).
+	 */
+	public function initial_notification_count(): int {
+		$count       = 0;
+		$preferences = get_user_meta( get_current_user_id(), 'wp_persisted_preferences', true );
+
+		if ( is_array( $preferences ) && isset( $preferences['woocommerce/google-listings-and-ads']['pmax-improve-assets-banner']['hasRecommendations'] ) && $preferences['woocommerce/google-listings-and-ads']['pmax-improve-assets-banner']['hasRecommendations'] ) {
+			return ++$count;
```

Comment:
```suggestion
			return 1;
```

Instead of increment `$count` can we just return 1 or 0? These way we don;t need `$count` variable

Reply:
IMO it's better to keep it as is because we may need some code block above this which may increase the count in that case, it will make sense to increment count instead of returning hardcoded value.